web_responsive/views/web.xml

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+
+    <template id="webclient_bootstrap"
+              inherit_id="web.webclient_bootstrap"
+              name="App Drawer - Web Client"
+              >
+
+        <xpath expr="//div[@class='o_sub_menu']" position="replace" />
+
+        <xpath expr="//t[@t-set='head']" position="inside">
+            <meta charset="utf-8" />
+            <meta http-equiv="cleartype" content="on" />
+            <meta name="MobileOptimized" content="320" />
+            <meta name="HandheldFriendly" content="True" />
+            <meta name="apple-mobile-web-app-capable" content="yes" />
+            <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
+        </xpath>
+
+        <xpath expr="//nav[@id='oe_main_menu_navbar']" position="replace">
+
+            <t t-set="body_classname" t-value="'drawer drawer--left o_web_client'" />
+
+            <header role="banner">
+                <nav id="odooAppDrawer" class="app-drawer-nav drawer-nav" role="navigation">
+                    <t t-call="web.menu" />
+                </nav>
+
+                <nav class="navbar navbar-default main-nav"
+                     role="navigation"
+                     groups="base.group_user,base.group_portal"
+                     >
+                    <div class="container-fluid">
+
+                        <div class="navbar-header">
+
+                            <a href="#"
```

Comment:
Could you please remove this href? It's creating a history entry each time you click on the drawer, and I think it will work equally without it...